Julia Version 1.3.2-pre.0
Commit 2e6715c045 (2019-12-31 00:49 UTC)
Platform Info:
  OS: Linux (x86_64-linux-gnu)
  CPU: Intel(R) Xeon(R) Silver 4114 CPU @ 2.20GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, skylake)
Environment:
  JULIA_DEPOT_PATH = ::/usr/local/share/julia

 Resolving package versions...
 Installed Mocking ───────────────────── v0.7.0
 Installed Tables ────────────────────── v0.2.11
 Installed OffsetArrays ──────────────── v0.11.4
 Installed IteratorInterfaceExtensions ─ v1.0.0
 Installed LibPQ ─────────────────────── v1.0.6
 Installed BinaryProvider ────────────── v0.5.8
 Installed Memento ───────────────────── v0.12.1
 Installed TimeZones ─────────────────── v0.10.3
 Installed CEnum ─────────────────────── v0.2.0
 Installed Parsers ───────────────────── v0.3.10
 Installed IterTools ─────────────────── v1.3.0
 Installed TableTraits ───────────────── v1.0.0
 Installed Syslogs ───────────────────── v0.3.0
 Installed LayerDicts ────────────────── v1.0.0
 Installed JSON ──────────────────────── v0.21.0
 Installed DataValueInterfaces ───────── v1.0.0
 Installed Decimals ──────────────────── v0.4.0
 Installed DataAPI ───────────────────── v1.1.0
 Installed DocStringExtensions ───────── v0.8.1
 Installed EzXML ─────────────────────── v0.9.5
  Updating `~/.julia/environments/v1.3/Project.toml`
  [194296ae] + LibPQ v1.0.6
  Updating `~/.julia/environments/v1.3/Manifest.toml`
  [b99e7846] + BinaryProvider v0.5.8
  [fa961155] + CEnum v0.2.0
  [9a962f9c] + DataAPI v1.1.0
  [e2d170a0] + DataValueInterfaces v1.0.0
  [abce61dc] + Decimals v0.4.0
  [ffbed154] + DocStringExtensions v0.8.1
  [8f5d6c58] + EzXML v0.9.5
  [c8e1da08] + IterTools v1.3.0
  [82899510] + IteratorInterfaceExtensions v1.0.0
  [682c06a0] + JSON v0.21.0
  [6f188dcb] + LayerDicts v1.0.0
  [194296ae] + LibPQ v1.0.6
  [f28f55f0] + Memento v0.12.1
  [78c3b35d] + Mocking v0.7.0
  [6fe1bfb0] + OffsetArrays v0.11.4
  [69de0a69] + Parsers v0.3.10
  [cea106d9] + Syslogs v0.3.0
  [3783bdb8] + TableTraits v1.0.0
  [bd369af6] + Tables v0.2.11
  [f269a46b] + TimeZones v0.10.3
  [2a0f44e3] + Base64 
  [ade2ca70] + Dates 
  [8ba89e20] + Distributed 
  [7b1f6079] + FileWatching 
  [b77e0a4c] + InteractiveUtils 
  [76f85450] + LibGit2 
  [8f399da3] + Libdl 
  [37e2e46d] + LinearAlgebra 
  [56ddb016] + Logging 
  [d6f4376e] + Markdown 
  [a63ad114] + Mmap 
  [44cfe95a] + Pkg 
  [de0858da] + Printf 
  [3fa0cd96] + REPL 
  [9a3f8284] + Random 
  [ea8e919c] + SHA 
  [9e88b42a] + Serialization 
  [6462fe0b] + Sockets 
  [8dfed614] + Test 
  [cf7118a7] + UUIDs 
  [4ec0a83e] + Unicode 
  Building EzXML ────→ `~/.julia/packages/EzXML/QtGgF/deps/build.log`
  Building TimeZones → `~/.julia/packages/TimeZones/pjvlM/deps/build.log`
  Building LibPQ ────→ `~/.julia/packages/LibPQ/PeS3p/deps/build.log`
   Testing LibPQ
 Resolving package versions...
 Installed SortingAlgorithms ── v0.3.1
 Installed Reexport ─────────── v0.2.0
 Installed DataStructures ───── v0.17.7
 Installed Compat ───────────── v3.2.0
 Installed InvertedIndices ──── v1.0.0
 Installed DataFrames ───────── v0.20.0
 Installed CategoricalArrays ── v0.7.6
 Installed Missings ─────────── v0.4.3
 Installed OrderedCollections ─ v1.1.0
 Installed PooledArrays ─────── v0.5.3
    Status `/tmp/jl_gzTnUJ/Manifest.toml`
  [b99e7846] BinaryProvider v0.5.8
  [fa961155] CEnum v0.2.0
  [324d7699] CategoricalArrays v0.7.6
  [34da2185] Compat v3.2.0
  [9a962f9c] DataAPI v1.1.0
  [a93c6f00] DataFrames v0.20.0
  [864edb3b] DataStructures v0.17.7
  [e2d170a0] DataValueInterfaces v1.0.0
  [abce61dc] Decimals v0.4.0
  [ffbed154] DocStringExtensions v0.8.1
  [8f5d6c58] EzXML v0.9.5
  [41ab1584] InvertedIndices v1.0.0
  [c8e1da08] IterTools v1.3.0
  [82899510] IteratorInterfaceExtensions v1.0.0
  [682c06a0] JSON v0.21.0
  [6f188dcb] LayerDicts v1.0.0
  [194296ae] LibPQ v1.0.6
  [f28f55f0] Memento v0.12.1
  [e1d29d7a] Missings v0.4.3
  [78c3b35d] Mocking v0.7.0
  [6fe1bfb0] OffsetArrays v0.11.4
  [bac558e1] OrderedCollections v1.1.0
  [69de0a69] Parsers v0.3.10
  [2dfb63ee] PooledArrays v0.5.3
  [189a3867] Reexport v0.2.0
  [a2af1166] SortingAlgorithms v0.3.1
  [cea106d9] Syslogs v0.3.0
  [3783bdb8] TableTraits v1.0.0
  [bd369af6] Tables v0.2.11
  [f269a46b] TimeZones v0.10.3
  [2a0f44e3] Base64  [`@stdlib/Base64`]
  [ade2ca70] Dates  [`@stdlib/Dates`]
  [8bb1440f] DelimitedFiles  [`@stdlib/DelimitedFiles`]
  [8ba89e20] Distributed  [`@stdlib/Distributed`]
  [7b1f6079] FileWatching  [`@stdlib/FileWatching`]
  [9fa8497b] Future  [`@stdlib/Future`]
  [b77e0a4c] InteractiveUtils  [`@stdlib/InteractiveUtils`]
  [76f85450] LibGit2  [`@stdlib/LibGit2`]
  [8f399da3] Libdl  [`@stdlib/Libdl`]
  [37e2e46d] LinearAlgebra  [`@stdlib/LinearAlgebra`]
  [56ddb016] Logging  [`@stdlib/Logging`]
  [d6f4376e] Markdown  [`@stdlib/Markdown`]
  [a63ad114] Mmap  [`@stdlib/Mmap`]
  [44cfe95a] Pkg  [`@stdlib/Pkg`]
  [de0858da] Printf  [`@stdlib/Printf`]
  [3fa0cd96] REPL  [`@stdlib/REPL`]
  [9a3f8284] Random  [`@stdlib/Random`]
  [ea8e919c] SHA  [`@stdlib/SHA`]
  [9e88b42a] Serialization  [`@stdlib/Serialization`]
  [1a1011a3] SharedArrays  [`@stdlib/SharedArrays`]
  [6462fe0b] Sockets  [`@stdlib/Sockets`]
  [2f01184e] SparseArrays  [`@stdlib/SparseArrays`]
  [10745b16] Statistics  [`@stdlib/Statistics`]
  [8dfed614] Test  [`@stdlib/Test`]
  [cf7118a7] UUIDs  [`@stdlib/UUIDs`]
  [4ec0a83e] Unicode  [`@stdlib/Unicode`]
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:76
  Expression: status(conn) == LibPQ.libpq_c.CONNECTION_OK
   Evaluated: LibPQ.libpq_c.CONNECTION_BAD == LibPQ.libpq_c.CONNECTION_OK
Stacktrace:
 [1] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:76
 [2] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [3] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
 [4] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
 [6] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:89
  Expression: status(result) == LibPQ.libpq_c.PGRES_TUPLES_OK
   Evaluated: LibPQ.libpq_c.PGRES_FATAL_ERROR == LibPQ.libpq_c.PGRES_TUPLES_OK
Stacktrace:
 [1] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:89
 [2] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [3] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
 [4] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
 [6] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:90
  Expression: isopen(result)
Stacktrace:
 [1] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:90
 [2] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [3] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
 [4] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
 [6] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:91
  Expression: LibPQ.num_columns(result) == 1
   Evaluated: 0 == 1
Stacktrace:
 [1] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:91
 [2] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [3] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
 [4] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
 [6] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:92
  Expression: LibPQ.num_rows(result) == 1
   Evaluated: 0 == 1
Stacktrace:
 [1] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:92
 [2] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [3] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
 [4] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
 [6] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
Example SELECT: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:93
  Test threw exception
  Expression: LibPQ.column_name(result, 1) == "typname"
  BoundsError: attempt to access 0-element Array{String,1} at index [1]
  Stacktrace:
   [1] getindex at ./array.jl:744 [inlined]
   [2] column_name(::LibPQ.Result, ::Int64) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/results.jl:433
   [3] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:93
   [4] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
   [6] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [8] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [9] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
  
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:94
  Expression: LibPQ.column_number(result, "typname") == 1
   Evaluated: 0 == 1
Stacktrace:
 [1] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:94
 [2] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [3] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
 [4] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
 [6] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
Example SELECT: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:98
  Test threw exception
  Expression: (data[:typname])[1] == "bool"
  type NamedTuple has no field typname
  Stacktrace:
   [1] getindex(::NamedTuple{(),Tuple{}}, ::Symbol) at ./namedtuple.jl:107
   [2] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:98
   [3] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
   [5] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
  
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:111
  Expression: status(result) == LibPQ.libpq_c.PGRES_TUPLES_OK
   Evaluated: LibPQ.libpq_c.PGRES_FATAL_ERROR == LibPQ.libpq_c.PGRES_TUPLES_OK
Stacktrace:
 [1] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:111
 [2] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [3] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
 [4] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
 [6] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:112
  Expression: isopen(result)
Stacktrace:
 [1] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:112
 [2] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [3] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
 [4] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
 [6] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:113
  Expression: LibPQ.num_columns(result) == 1
   Evaluated: 0 == 1
Stacktrace:
 [1] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:113
 [2] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [3] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
 [4] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
 [6] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:114
  Expression: LibPQ.num_rows(result) == 1
   Evaluated: 0 == 1
Stacktrace:
 [1] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:114
 [2] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [3] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
 [4] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
 [6] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
Example SELECT: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:115
  Test threw exception
  Expression: LibPQ.column_name(result, 1) == "typname"
  BoundsError: attempt to access 0-element Array{String,1} at index [1]
  Stacktrace:
   [1] getindex at ./array.jl:744 [inlined]
   [2] column_name(::LibPQ.Result, ::Int64) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/results.jl:433
   [3] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:115
   [4] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
   [6] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [8] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [9] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
  
Example SELECT: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:119
  Test threw exception
  Expression: (data[:typname])[1] == "bool"
  type NamedTuple has no field typname
  Stacktrace:
   [1] getindex(::NamedTuple{(),Tuple{}}, ::Symbol) at ./namedtuple.jl:107
   [2] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:119
   [3] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
   [5] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
  
Example SELECT: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:72
  Got exception outside of a @test
  UnknownError: 
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQResultError{LibPQ.Errors.CUN,LibPQ.Errors.EUNOWN}) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_result#51(::Bool, ::typeof(LibPQ.handle_result), ::LibPQ.Result) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/results.jl:226
   [3] #handle_result at ./none:0 [inlined]
   [4] prepare(::LibPQ.Connection, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/statements.jl:48
   [5] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:127
   [6] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [7] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:73
   [8] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [9] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [10] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [11] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [12] include at ./boot.jl:328 [inlined]
   [13] include_relative(::Module, ::String) at ./loading.jl:1105
   [14] include(::Module, ::String) at ./Base.jl:31
   [15] include(::String) at ./client.jl:424
   [16] top-level scope at none:6
   [17] eval(::Module, ::Any) at ./boot.jl:330
   [18] exec_options(::Base.JLOptions) at ./client.jl:263
   [19] _start() at ./client.jl:460
  
Example INSERT and DELETE: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:180
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] LibPQ.Connection(::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:272
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:181
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:181
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [13] include at ./boot.jl:328 [inlined]
   [14] include_relative(::Module, ::String) at ./loading.jl:1105
   [15] include(::Module, ::String) at ./Base.jl:31
   [16] include(::String) at ./client.jl:424
   [17] top-level scope at none:6
   [18] eval(::Module, ::Any) at ./boot.jl:330
   [19] exec_options(::Base.JLOptions) at ./client.jl:263
   [20] _start() at ./client.jl:460
  
Example COPY FROM: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:277
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] LibPQ.Connection(::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:272
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:278
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:278
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:277
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [15] include at ./boot.jl:328 [inlined]
   [16] include_relative(::Module, ::String) at ./loading.jl:1105
   [17] include(::Module, ::String) at ./Base.jl:31
   [18] include(::String) at ./client.jl:424
   [19] top-level scope at none:6
   [20] eval(::Module, ::Any) at ./boot.jl:330
   [21] exec_options(::Base.JLOptions) at ./client.jl:263
   [22] _start() at ./client.jl:460
  
Wrong column order: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:321
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] LibPQ.Connection(::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:272
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:322
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:322
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:277
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [15] include at ./boot.jl:328 [inlined]
   [16] include_relative(::Module, ::String) at ./loading.jl:1105
   [17] include(::Module, ::String) at ./Base.jl:31
   [18] include(::String) at ./client.jl:424
   [19] top-level scope at none:6
   [20] eval(::Module, ::Any) at ./boot.jl:330
   [21] exec_options(::Base.JLOptions) at ./client.jl:263
   [22] _start() at ./client.jl:460
  
do: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:376
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] Type at ./none:0 [inlined]
   [6] #Connection#24(::Base.Iterators.Pairs{Symbol,Bool,Tuple{Symbol},NamedTuple{(:throw_error,),Tuple{Bool}}}, ::Type{LibPQ.Connection}, ::var"#6#35", ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:350
   [7] (::Core.var"#kw#Type")(::NamedTuple{(:throw_error,),Tuple{Bool}}, ::Type{LibPQ.Connection}, ::Function, ::String) at ./none:0
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:379
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:377
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:376
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [15] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [16] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [17] include at ./boot.jl:328 [inlined]
   [18] include_relative(::Module, ::String) at ./loading.jl:1105
   [19] include(::Module, ::String) at ./Base.jl:31
   [20] include(::String) at ./client.jl:424
   [21] top-level scope at none:6
   [22] eval(::Module, ::Any) at ./boot.jl:330
   [23] exec_options(::Base.JLOptions) at ./client.jl:263
   [24] _start() at ./client.jl:460
  
Version Numbers: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:395
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] (::Core.var"#kw#Type")(::NamedTuple{(:throw_error,),Tuple{Bool}}, ::Type{LibPQ.Connection}, ::String) at ./none:0
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:396
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:396
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:376
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [15] include at ./boot.jl:328 [inlined]
   [16] include_relative(::Module, ::String) at ./loading.jl:1105
   [17] include(::Module, ::String) at ./Base.jl:31
   [18] include(::String) at ./client.jl:424
   [19] top-level scope at none:6
   [20] eval(::Module, ::Any) at ./boot.jl:330
   [21] exec_options(::Base.JLOptions) at ./client.jl:263
   [22] _start() at ./client.jl:460
  
Encoding: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:402
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] (::Core.var"#kw#Type")(::NamedTuple{(:throw_error,),Tuple{Bool}}, ::Type{LibPQ.Connection}, ::String) at ./none:0
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:403
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:403
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:376
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [15] include at ./boot.jl:328 [inlined]
   [16] include_relative(::Module, ::String) at ./loading.jl:1105
   [17] include(::Module, ::String) at ./Base.jl:31
   [18] include(::String) at ./client.jl:424
   [19] top-level scope at none:6
   [20] eval(::Module, ::Any) at ./boot.jl:330
   [21] exec_options(::Base.JLOptions) at ./client.jl:263
   [22] _start() at ./client.jl:460
  
Options: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:428
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Symbol,Dict{Symbol,DataType},Tuple{Symbol},NamedTuple{(:type_map,),Tuple{Dict{Symbol,DataType}}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] (::Core.var"#kw#Type")(::NamedTuple{(:options, :throw_error, :type_map),Tuple{Dict{String,String},Bool,Dict{Symbol,DataType}}}, ::Type{LibPQ.Connection}, ::String) at ./none:0
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:429
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:429
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:376
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [15] include at ./boot.jl:328 [inlined]
   [16] include_relative(::Module, ::String) at ./loading.jl:1105
   [17] include(::Module, ::String) at ./Base.jl:31
   [18] include(::String) at ./client.jl:424
   [19] top-level scope at none:6
   [20] eval(::Module, ::Any) at ./boot.jl:330
   [21] exec_options(::Base.JLOptions) at ./client.jl:263
   [22] _start() at ./client.jl:460
  
Time Zone: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:447
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] Type at ./none:0 [inlined]
   [6] #Connection#24(::Base.Iterators.Pairs{Symbol,Bool,Tuple{Symbol},NamedTuple{(:throw_error,),Tuple{Bool}}}, ::Type{LibPQ.Connection}, ::var"#10#40"{var"#connection_tz#38"}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:350
   [7] Type at ./none:0 [inlined]
   [8] (::var"#9#39"{String,var"#connection_tz#38"})() at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:464
   [9] withenv(::var"#9#39"{String,var"#connection_tz#38"}, ::Pair{String,Nothing}) at ./env.jl:161
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:463
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:448
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:376
   [15] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [16] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [17] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [18] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [19] include at ./boot.jl:328 [inlined]
   [20] include_relative(::Module, ::String) at ./loading.jl:1105
   [21] include(::Module, ::String) at ./Base.jl:31
   [22] include(::String) at ./client.jl:424
   [23] top-level scope at none:6
   [24] eval(::Module, ::Any) at ./boot.jl:330
   [25] exec_options(::Base.JLOptions) at ./client.jl:263
   [26] _start() at ./client.jl:460
  
Finalizer: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:548
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] Connection at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:272 [inlined]
   [6] #21 at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:550 [inlined]
   [7] iterate at ./generator.jl:47 [inlined]
   [8] _collect(::UnitRange{Int64}, ::Base.Generator{UnitRange{Int64},var"#21#51"{String}}, ::Base.EltypeUnknown, ::Base.HasShape{1}) at ./array.jl:635
   [9] collect_similar(::UnitRange{Int64}, ::Base.Generator{UnitRange{Int64},var"#21#51"{String}}) at ./array.jl:564
   [10] map(::Function, ::UnitRange{Int64}) at ./abstractarray.jl:2073
   [11] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:549
   [12] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [13] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:549
   [14] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [15] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:376
   [16] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [17] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [18] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [19] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [20] include at ./boot.jl:328 [inlined]
   [21] include_relative(::Module, ::String) at ./loading.jl:1105
   [22] include(::Module, ::String) at ./Base.jl:31
   [23] include(::String) at ./client.jl:424
   [24] top-level scope at none:6
   [25] eval(::Module, ::Any) at ./boot.jl:330
   [26] exec_options(::Base.JLOptions) at ./client.jl:263
   [27] _start() at ./client.jl:460
  
closed socket (EBADF): Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:609
  Got exception outside of a @test
  SystemError: fdopen: Bad file descriptor
  Stacktrace:
   [1] #systemerror#44(::Nothing, ::typeof(systemerror), ::String, ::Bool) at ./error.jl:134
   [2] systemerror at ./error.jl:134 [inlined]
   [3] Base.Libc.FILE(::RawFD, ::String) at ./libc.jl:82
   [4] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:611
   [5] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:610
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:593
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:376
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [15] include at ./boot.jl:328 [inlined]
   [16] include_relative(::Module, ::String) at ./loading.jl:1105
   [17] include(::Module, ::String) at ./Base.jl:31
   [18] include(::String) at ./client.jl:424
   [19] top-level scope at none:6
   [20] eval(::Module, ::Any) at ./boot.jl:330
   [21] exec_options(::Base.JLOptions) at ./client.jl:263
   [22] _start() at ./client.jl:460
  
Nulls: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] (::Core.var"#kw#Type")(::NamedTuple{(:throw_error,),Tuple{Bool}}, ::Type{LibPQ.Connection}, ::String) at ./none:0
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:658
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:658
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [15] include at ./boot.jl:328 [inlined]
   [16] include_relative(::Module, ::String) at ./loading.jl:1105
   [17] include(::Module, ::String) at ./Base.jl:31
   [18] include(::String) at ./client.jl:424
   [19] top-level scope at none:6
   [20] eval(::Module, ::Any) at ./boot.jl:330
   [21] exec_options(::Base.JLOptions) at ./client.jl:263
   [22] _start() at ./client.jl:460
  
Not Nulls: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:741
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] (::Core.var"#kw#Type")(::NamedTuple{(:throw_error,),Tuple{Bool}}, ::Type{LibPQ.Connection}, ::String) at ./none:0
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:742
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:742
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [15] include at ./boot.jl:328 [inlined]
   [16] include_relative(::Module, ::String) at ./loading.jl:1105
   [17] include(::Module, ::String) at ./Base.jl:31
   [18] include(::String) at ./client.jl:424
   [19] top-level scope at none:6
   [20] eval(::Module, ::Any) at ./boot.jl:330
   [21] exec_options(::Base.JLOptions) at ./client.jl:263
   [22] _start() at ./client.jl:460
  
Tables.jl: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:820
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] (::Core.var"#kw#Type")(::NamedTuple{(:throw_error,),Tuple{Bool}}, ::Type{LibPQ.Connection}, ::String) at ./none:0
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:821
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:821
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [15] include at ./boot.jl:328 [inlined]
   [16] include_relative(::Module, ::String) at ./loading.jl:1105
   [17] include(::Module, ::String) at ./Base.jl:31
   [18] include(::String) at ./client.jl:424
   [19] top-level scope at none:6
   [20] eval(::Module, ::Any) at ./boot.jl:330
   [21] exec_options(::Base.JLOptions) at ./client.jl:263
   [22] _start() at ./client.jl:460
  
Duplicate names: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:862
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] (::Core.var"#kw#Type")(::NamedTuple{(:throw_error,),Tuple{Bool}}, ::Type{LibPQ.Connection}, ::String) at ./none:0
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:863
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:863
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [15] include at ./boot.jl:328 [inlined]
   [16] include_relative(::Module, ::String) at ./loading.jl:1105
   [17] include(::Module, ::String) at ./Base.jl:31
   [18] include(::String) at ./client.jl:424
   [19] top-level scope at none:6
   [20] eval(::Module, ::Any) at ./boot.jl:330
   [21] exec_options(::Base.JLOptions) at ./client.jl:263
   [22] _start() at ./client.jl:460
  
Uppercase Columns: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:878
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] (::Core.var"#kw#Type")(::NamedTuple{(:throw_error,),Tuple{Bool}}, ::Type{LibPQ.Connection}, ::String) at ./none:0
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:879
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:879
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [15] include at ./boot.jl:328 [inlined]
   [16] include_relative(::Module, ::String) at ./loading.jl:1105
   [17] include(::Module, ::String) at ./Base.jl:31
   [18] include(::String) at ./client.jl:424
   [19] top-level scope at none:6
   [20] eval(::Module, ::Any) at ./boot.jl:330
   [21] exec_options(::Base.JLOptions) at ./client.jl:263
   [22] _start() at ./client.jl:460
  
PQResultError: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:896
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] (::Core.var"#kw#Type")(::NamedTuple{(:throw_error,),Tuple{Bool}}, ::Type{LibPQ.Connection}, ::String) at ./none:0
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:897
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:897
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [15] include at ./boot.jl:328 [inlined]
   [16] include_relative(::Module, ::String) at ./loading.jl:1105
   [17] include(::Module, ::String) at ./Base.jl:31
   [18] include(::String) at ./client.jl:424
   [19] top-level scope at none:6
   [20] eval(::Module, ::Any) at ./boot.jl:330
   [21] exec_options(::Base.JLOptions) at ./client.jl:263
   [22] _start() at ./client.jl:460
  
Automatic: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:943
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] (::Core.var"#kw#Type")(::NamedTuple{(:throw_error,),Tuple{Bool}}, ::Type{LibPQ.Connection}, ::String) at ./none:0
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:944
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:944
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:943
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [15] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [16] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [17] include at ./boot.jl:328 [inlined]
   [18] include_relative(::Module, ::String) at ./loading.jl:1105
   [19] include(::Module, ::String) at ./Base.jl:31
   [20] include(::String) at ./client.jl:424
   [21] top-level scope at none:6
   [22] eval(::Module, ::Any) at ./boot.jl:330
   [23] exec_options(::Base.JLOptions) at ./client.jl:263
   [24] _start() at ./client.jl:460
  
Overrides: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:974
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] (::Core.var"#kw#Type")(::NamedTuple{(:throw_error,),Tuple{Bool}}, ::Type{LibPQ.Connection}, ::String) at ./none:0
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:975
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:975
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:943
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [15] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [16] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [17] include at ./boot.jl:328 [inlined]
   [18] include_relative(::Module, ::String) at ./loading.jl:1105
   [19] include(::Module, ::String) at ./Base.jl:31
   [20] include(::String) at ./client.jl:424
   [21] top-level scope at none:6
   [22] eval(::Module, ::Any) at ./boot.jl:330
   [23] exec_options(::Base.JLOptions) at ./client.jl:263
   [24] _start() at ./client.jl:460
  
Default Types: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1013
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] (::Core.var"#kw#Type")(::NamedTuple{(:throw_error,),Tuple{Bool}}, ::Type{LibPQ.Connection}, ::String) at ./none:0
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1014
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1014
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1013
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:943
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
   [15] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [16] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [17] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [18] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [19] include at ./boot.jl:328 [inlined]
   [20] include_relative(::Module, ::String) at ./loading.jl:1105
   [21] include(::Module, ::String) at ./Base.jl:31
   [22] include(::String) at ./client.jl:424
   [23] top-level scope at none:6
   [24] eval(::Module, ::Any) at ./boot.jl:330
   [25] exec_options(::Base.JLOptions) at ./client.jl:263
   [26] _start() at ./client.jl:460
  
Specified Types: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1125
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] (::Core.var"#kw#Type")(::NamedTuple{(:throw_error,),Tuple{Bool}}, ::Type{LibPQ.Connection}, ::String) at ./none:0
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1126
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1126
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1013
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:943
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
   [15] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [16] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [17] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [18] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [19] include at ./boot.jl:328 [inlined]
   [20] include_relative(::Module, ::String) at ./loading.jl:1105
   [21] include(::Module, ::String) at ./Base.jl:31
   [22] include(::String) at ./client.jl:424
   [23] top-level scope at none:6
   [24] eval(::Module, ::Any) at ./boot.jl:330
   [25] exec_options(::Base.JLOptions) at ./client.jl:263
   [26] _start() at ./client.jl:460
  
Parameters: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1187
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] (::Core.var"#kw#Type")(::NamedTuple{(:throw_error,),Tuple{Bool}}, ::Type{LibPQ.Connection}, ::String) at ./none:0
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1188
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1188
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [15] include at ./boot.jl:328 [inlined]
   [16] include_relative(::Module, ::String) at ./loading.jl:1105
   [17] include(::Module, ::String) at ./Base.jl:31
   [18] include(::String) at ./client.jl:424
   [19] top-level scope at none:6
   [20] eval(::Module, ::Any) at ./boot.jl:330
   [21] exec_options(::Base.JLOptions) at ./client.jl:263
   [22] _start() at ./client.jl:460
  
Syntax Errors: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1222
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] (::Core.var"#kw#Type")(::NamedTuple{(:throw_error,),Tuple{Bool}}, ::Type{LibPQ.Connection}, ::String) at ./none:0
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1223
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1223
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1222
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [15] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [16] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [17] include at ./boot.jl:328 [inlined]
   [18] include_relative(::Module, ::String) at ./loading.jl:1105
   [19] include(::Module, ::String) at ./Base.jl:31
   [20] include(::String) at ./client.jl:424
   [21] top-level scope at none:6
   [22] eval(::Module, ::Any) at ./boot.jl:330
   [23] exec_options(::Base.JLOptions) at ./client.jl:263
   [24] _start() at ./client.jl:460
  
Wrong No. Parameters: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1240
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] (::Core.var"#kw#Type")(::NamedTuple{(:throw_error,),Tuple{Bool}}, ::Type{LibPQ.Connection}, ::String) at ./none:0
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1241
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1241
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1222
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [15] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [16] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [17] include at ./boot.jl:328 [inlined]
   [18] include_relative(::Module, ::String) at ./loading.jl:1105
   [19] include(::Module, ::String) at ./Base.jl:31
   [20] include(::String) at ./client.jl:424
   [21] top-level scope at none:6
   [22] eval(::Module, ::Any) at ./boot.jl:330
   [23] exec_options(::Base.JLOptions) at ./client.jl:263
   [24] _start() at ./client.jl:460
  
Interface Errors: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1264
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] (::Core.var"#kw#Type")(::NamedTuple{(:throw_error,),Tuple{Bool}}, ::Type{LibPQ.Connection}, ::String) at ./none:0
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1265
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1265
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:657
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [15] include at ./boot.jl:328 [inlined]
   [16] include_relative(::Module, ::String) at ./loading.jl:1105
   [17] include(::Module, ::String) at ./Base.jl:31
   [18] include(::String) at ./client.jl:424
   [19] top-level scope at none:6
   [20] eval(::Module, ::Any) at ./boot.jl:330
   [21] exec_options(::Base.JLOptions) at ./client.jl:263
   [22] _start() at ./client.jl:460
  
No Params, Output: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1281
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] (::Core.var"#kw#Type")(::NamedTuple{(:throw_error,),Tuple{Bool}}, ::Type{LibPQ.Connection}, ::String) at ./none:0
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1282
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1282
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1281
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [15] include at ./boot.jl:328 [inlined]
   [16] include_relative(::Module, ::String) at ./loading.jl:1105
   [17] include(::Module, ::String) at ./Base.jl:31
   [18] include(::String) at ./client.jl:424
   [19] top-level scope at none:6
   [20] eval(::Module, ::Any) at ./boot.jl:330
   [21] exec_options(::Base.JLOptions) at ./client.jl:263
   [22] _start() at ./client.jl:460
  
Params, Output: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1302
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] (::Core.var"#kw#Type")(::NamedTuple{(:throw_error,),Tuple{Bool}}, ::Type{LibPQ.Connection}, ::String) at ./none:0
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1303
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1303
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1281
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [15] include at ./boot.jl:328 [inlined]
   [16] include_relative(::Module, ::String) at ./loading.jl:1105
   [17] include(::Module, ::String) at ./Base.jl:31
   [18] include(::String) at ./client.jl:424
   [19] top-level scope at none:6
   [20] eval(::Module, ::Any) at ./boot.jl:330
   [21] exec_options(::Base.JLOptions) at ./client.jl:263
   [22] _start() at ./client.jl:460
  
Basic: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1331
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] (::Core.var"#kw#Type")(::NamedTuple{(:throw_error,),Tuple{Bool}}, ::Type{LibPQ.Connection}, ::String) at ./none:0
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1332
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1332
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1329
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [15] include at ./boot.jl:328 [inlined]
   [16] include_relative(::Module, ::String) at ./loading.jl:1105
   [17] include(::Module, ::String) at ./Base.jl:31
   [18] include(::String) at ./client.jl:424
   [19] top-level scope at none:6
   [20] eval(::Module, ::Any) at ./boot.jl:330
   [21] exec_options(::Base.JLOptions) at ./client.jl:263
   [22] _start() at ./client.jl:460
  
Parameters: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1353
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] (::Core.var"#kw#Type")(::NamedTuple{(:throw_error,),Tuple{Bool}}, ::Type{LibPQ.Connection}, ::String) at ./none:0
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1354
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1354
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1329
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [15] include at ./boot.jl:328 [inlined]
   [16] include_relative(::Module, ::String) at ./loading.jl:1105
   [17] include(::Module, ::String) at ./Base.jl:31
   [18] include(::String) at ./client.jl:424
   [19] top-level scope at none:6
   [20] eval(::Module, ::Any) at ./boot.jl:330
   [21] exec_options(::Base.JLOptions) at ./client.jl:263
   [22] _start() at ./client.jl:460
  
Wait in line to complete: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1385
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] (::Core.var"#kw#Type")(::NamedTuple{(:throw_error,),Tuple{Bool}}, ::Type{LibPQ.Connection}, ::String) at ./none:0
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1386
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1386
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1329
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [15] include at ./boot.jl:328 [inlined]
   [16] include_relative(::Module, ::String) at ./loading.jl:1105
   [17] include(::Module, ::String) at ./Base.jl:31
   [18] include(::String) at ./client.jl:424
   [19] top-level scope at none:6
   [20] eval(::Module, ::Any) at ./boot.jl:330
   [21] exec_options(::Base.JLOptions) at ./client.jl:263
   [22] _start() at ./client.jl:460
  
Cancel: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1414
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] (::Core.var"#kw#Type")(::NamedTuple{(:throw_error,),Tuple{Bool}}, ::Type{LibPQ.Connection}, ::String) at ./none:0
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1415
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1415
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1329
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [15] include at ./boot.jl:328 [inlined]
   [16] include_relative(::Module, ::String) at ./loading.jl:1105
   [17] include(::Module, ::String) at ./Base.jl:31
   [18] include(::String) at ./client.jl:424
   [19] top-level scope at none:6
   [20] eval(::Module, ::Any) at ./boot.jl:330
   [21] exec_options(::Base.JLOptions) at ./client.jl:263
   [22] _start() at ./client.jl:460
  
Canceled by closing connection: Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1446
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] (::Core.var"#kw#Type")(::NamedTuple{(:throw_error,),Tuple{Bool}}, ::Type{LibPQ.Connection}, ::String) at ./none:0
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1447
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1447
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1329
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [15] include at ./boot.jl:328 [inlined]
   [16] include_relative(::Module, ::String) at ./loading.jl:1105
   [17] include(::Module, ::String) at ./Base.jl:31
   [18] include(::String) at ./client.jl:424
   [19] top-level scope at none:6
   [20] eval(::Module, ::Any) at ./boot.jl:330
   [21] exec_options(::Base.JLOptions) at ./client.jl:263
   [22] _start() at ./client.jl:460
  
FDWatcher: bad file descriptor (EBADF): Error During Test at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1478
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
   [1] error(::Logger, ::LibPQ.Errors.PQConnectionError) at /home/pkgeval/.julia/packages/Memento/UgJr2/src/loggers.jl:439
   [2] #handle_new_connection#9(::Bool, ::typeof(LibPQ.handle_new_connection), ::LibPQ.Connection) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:127
   [3] #handle_new_connection at ./none:0 [inlined]
   [4] #Connection#17(::Bool, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type{LibPQ.Connection}, ::String) at /home/pkgeval/.julia/packages/LibPQ/PeS3p/src/connections.jl:320
   [5] (::Core.var"#kw#Type")(::NamedTuple{(:throw_error,),Tuple{Bool}}, ::Type{LibPQ.Connection}, ::String) at ./none:0
   [6] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1479
   [7] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [8] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1479
   [9] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [10] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:1329
   [11] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [12] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:70
   [13] top-level scope at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [14] top-level scope at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:34
   [15] include at ./boot.jl:328 [inlined]
   [16] include_relative(::Module, ::String) at ./loading.jl:1105
   [17] include(::Module, ::String) at ./Base.jl:31
   [18] include(::String) at ./client.jl:424
   [19] top-level scope at none:6
   [20] eval(::Module, ::Any) at ./boot.jl:330
   [21] exec_options(::Base.JLOptions) at ./client.jl:263
   [22] _start() at ./client.jl:460
  
Test Summary:                                | Pass  Fail  Error  Total
LibPQ                                        |   51    10     37     98
  ConninfoDisplay                            |    4                   4
  Version Numbers                            |   11                  11
  Online                                     |   36    10     37     83
    Example SELECT                           |    9    10      5     24
    Example INSERT and DELETE                |                 1      1
    COPY FROM                                |                 2      2
      Example COPY FROM                      |                 1      1
      Wrong column order                     |                 1      1
    LibPQ.Connection                         |   18            7     25
      do                                     |                 1      1
      Version Numbers                        |                 1      1
      Encoding                               |                 1      1
      Options                                |                 1      1
      Time Zone                              |                 1      1
      Finalizer                              |                 1      1
      Bad Connection                         |   18            1     19
        timeout                              |    2                   2
        closed socket (EBADF)                |                 1      1
        throw_error=false                    |    8                   8
        throw_error=true                     |    8                   8
    Results                                  |    9           14     23
      Nulls                                  |                 1      1
      Not Nulls                              |                 1      1
      Tables.jl                              |                 1      1
      Duplicate names                        |                 1      1
      Uppercase Columns                      |                 1      1
      PQResultError                          |                 1      1
      Type Conversions                       |    9            4     13
        Automatic                            |                 1      1
        Overrides                            |                 1      1
        Parsing                              |    9            2     11
          Default Types                      |                 1      1
          Specified Types                    |                 1      1
          Interval Regex                     |    9                   9
      Parameters                             |                 1      1
      Query Errors                           |                 2      2
        Syntax Errors                        |                 1      1
        Wrong No. Parameters                 |                 1      1
      Interface Errors                       |                 1      1
    Statements                               |                 2      2
      No Params, Output                      |                 1      1
      Params, Output                         |                 1      1
    AsyncResults                             |                 6      6
      Basic                                  |                 1      1
      Parameters                             |                 1      1
      Wait in line to complete               |                 1      1
      Cancel                                 |                 1      1
      Canceled by closing connection         |                 1      1
      FDWatcher: bad file descriptor (EBADF) |                 1      1
ERROR: LoadError: Some tests did not pass: 51 passed, 10 failed, 37 errored, 0 broken.
in expression starting at /home/pkgeval/.julia/packages/LibPQ/PeS3p/test/runtests.jl:32
ERROR: Package LibPQ errored during testing
Stacktrace:
 [1] pkgerror(::String, ::Vararg{String,N} where N) at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Pkg/src/Types.jl:113
 [2] #test#131(::Bool, ::Nothing, ::Cmd, ::Cmd, ::typeof(Pkg.Operations.test), ::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}) at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Pkg/src/Operations.jl:1372
 [3] #test at ./none:0 [inlined]
 [4] #test#62(::Bool, ::Nothing, ::Cmd, ::Cmd, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::typeof(Pkg.API.test), ::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}) at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Pkg/src/API.jl:253
 [5] test(::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}) at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Pkg/src/API.jl:239
 [6] #test#61 at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Pkg/src/API.jl:233 [inlined]
 [7] test at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Pkg/src/API.jl:233 [inlined]
 [8] #test#60 at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Pkg/src/API.jl:232 [inlined]
 [9] test at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Pkg/src/API.jl:232 [inlined]
 [10] #test#59(::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::typeof(Pkg.API.test), ::String) at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Pkg/src/API.jl:231
 [11] test(::String) at /workspace/srcdir/usr/share/julia/stdlib/v1.3/Pkg/src/API.jl:231
 [12] top-level scope at none:13
